source/edge-server/connect.txt

```diff
@@ -170,58 +207,155 @@ modifications you can make to the :github:`Swift template app
+       ``ec2-13-58-70-88.us-east-2.compute.amazonaws.com``. For more 
+       information, refer to :ref:`edge-server-install-and-configure`.
 
-The ``EDGE_SERVER_HOSTNAME`` is the public DNS of your Edge Server host. 
-For more information, refer to :ref:`edge-server-install-and-configure`.
+   * - ``<port>``
+     - The Edge Server Wire Protocol listen port. The default port is ``27021``.
+       For more details, refer to :ref:`edge-server-install-and-configure`.
 
-You must make port 27021 accessible to the client through port mapping or
-firewall rules.
+   * - ``authMechanism=PLAIN``
+     - The auth mechanism. Currently, the only supported value is ``PLAIN``.
 
 .. example::
 
-   A wire protocol connection URI for an Edge Server hosted on Amazon EC2 
-   might look similar to:
+   A Wire Protocol connection URI for an Edge Server instance hosted on Amazon 
+   EC2 might look similar to:
 
    .. code:: console
 
-      mongodb://ec2-13-58-70-88.us-east-2.compute.amazonaws.com:27021
+      mongodb://example-user:my-password@ec2-13-58-70-88.us-east-2.compute.amazonaws.com:27021?authMechanism=PLAIN
+
+.. _anonymous-connection-URI:
+
+Anonymous Connections
+~~~~~~~~~~~~~~~~~~~~~
+
+You can enable anonymous connections and connect without specifying a username 
+or password. This option is disabled by default.
+
+.. procedure::
+
+   .. step:: Configure Edge Server with the Relevant Flag
+
+      To enable anonymous connections, start the Edge Server with the
+      ``--insecure-disable-auth`` flag set to ``true``. This allows you to
+      connect without specifying a username and password.
+
+      This setting is ``false`` by default. Setting this to ``true`` enables
+      connections to bypass authentication.
+
+      For more information about using ``edgectl`` to configure and start
+      the Edge Server instance, refer to 
+      :ref:`edge-server-install-and-configure`.
+
+   .. step:: Omit Authentication Details from MongoDB URI
+
+      When connecting without authentication, the MongoDB URI omits the 
+      authentication-related parameters. This simplfied URI is:
+
+      .. code:: console
+
+         mongodb://<hostname>:<port>
+
+      .. example::
+
+         An anonymous Wire Protocol connection URI for an Edge Server instance 
+         hosted on Amazon EC2 might look similar to:
+
+         .. code:: console
+
+            mongodb://ec2-13-58-70-88.us-east-2.compute.amazonaws.com:27021
+        
+         An anonymous connection to an Edge Server instance running on your
+         machine during development may resemble:
+
+         .. code:: console
+
+            mongodb://localhost:27021
+
+Supported MongoDB APIs
+~~~~~~~~~~~~~~~~~~~~~~
 
 Edge Server currently supports a subset of the MongoDB APIs. For information 
 about the supported APIs, refer to :ref:`edge-server-mongodb`.
+
+.. _edge-server-compass-connect:
+
+Connect with MongoDB Compass
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Connect to the Edge Server with MongoDB Compass
-----------------------------------------------
```

Reply:
I initially had this as its own section. But it uses the same MongoDB URI to connect, so it didn't seem worth having another section for it and then referring people to this section for the connection string details. I left it in the intro because I wanted to call it out as a way to explore documents on the Edge Server, which seemed useful for folks who are in development and want to understand what's syncing to the Edge Server.